poi_fr.md

```diff
@@ -0,0 +1,211 @@
+sous-catégorie IGN | `ref:PDL:IGN:subcategory` | contenu du champ | modifier le trigramme PDL en cas de nouvelle source IGN
+source de la donnée (libellé) | `source` | |
+source de la données (id) | `ref:PDL:IGN:id` | | modifier le trigramme PDL en cas de nouvelle source IGN
+date de création de la donnée | `creation_date` | | 
+date de modification de la donnée | `modification_date` | | 
+Description | `description` | | 
+Adresse postale 1 | `addr:full` | | utiliser `addr:full` si le numéro est inclus dans l'adresse. Si le numéro est à part, utiliser le champ `addr:street` et `addr:housenumber`
+Code postal 1 | `addr:postcode` | | 
+Commune 1 | `addr:city` | | 
+Code insee 1 | `ref:INSEE` | | 
+Adresse postale 2 | `addr:full:2` | | utiliser `addr:full:2` si le numéro est inclus dans l'adresse. Si le numéro est à part, utiliser le champ `addr:street:2` et `addr:housenumber:2`
+Code postal 2 | `addr:postcode:2` | | 
+Commune 2 | `addr:city:2` | | 
+Code INSEE 2 | `ref:INSEE:2` | | 
+X2 | `x2` | | Coord X d'un 2ème accès en WGS84 décimal (séparateur ".")
+Y2 | `y2` | | Coord Y d'un 2ème accès en WGS84 décimal (séparateur ".")
+téléphone | `phone` | | 
+URL | `website` | | 
+Accessibilité | `access` | | 
+Commentaire (différent de description ???) | `comment` | | 
+Ouverture | `opening` | | la clé opening_hours a un format normalisée, pas possible de l'utiliser
+(transport) Capacité | `capacity` | texte libre | 
+(transport) Taux de remplissage | | | 
+(transport) Ouverture | | | `opening` est déjà présent plus haut, ce champ n'est a priori pas pertinent.
+(transport) condition d'accès | `access_condition` | |
+	
+Il existe également certaines informations qui ne sont pas paramétrables (à ce
+jour) dans les propriétés des POI :
+
+- le temps de prise et de repose d'un VLS (la valeur par défaut est à 2 minutes
+  pour la prise, et 1 minute pour la repose)
+- le temps de dépot et de reprise d'une voiture dans un parking (la valeur par
+  défaut est à 5 minutes)
+
+# Liste standard des catégories
+
+Les catégories dans Navitia sont standardisées en utilisant OSM comme source de
+données. Les POI pouvant avoir plusieurs "tags" utilisés comme source de
+catégories Navitia, ces tags sont priorisées de la manière suivante :
+
+- utilisation du tag `amenity` (lien vers la [description OSM](http://wiki.openstreetmap.org/wiki/Key:amenity))
+- utilisation du tag `tourism` (lien vers la [description OSM](http://wiki.openstreetmap.org/wiki/Key:tourism))
+- utilisation du tag `leisure` (lien vers la [description OSM](http://wiki.openstreetmap.org/wiki/Key:leisure))
+- utilisation du tag `aeroway` (lien vers la [description OSM](http://wiki.openstreetmap.org/wiki/Key:aeroway))
+- utilisation du tag `railway` (lien vers la [description OSM](http://wiki.openstreetmap.org/wiki/Key:railway))
+- utilisation du tag `shop` (lien vers la [description OSM](http://wiki.openstreetmap.org/wiki/Key:shop))
```

Comment:
I've removed a part here about a big SQL query that didn't make so much sense... Please yell at me if you think it needs to stay here. If yes, please help me with some comments about what is this query and why it is here.

Reply:
Outdated since this whole part has been removed.